stixcore/soc/manager.py

```diff
@@ -0,0 +1,86 @@
+import os
+from pathlib import Path
+from binascii import unhexlify
+from xml.etree import ElementTree as Et
+from collections import defaultdict
+
+from stixcore.tmtc.packets import TMTC, TMPacket
+from stixcore.util.logging import get_logger
+
+__all__ = ['SOCManager', 'SOCPacketFile']
+
+
+logger = get_logger(__name__)
+
+
+class SOCPacketFile:
+    def __init__(self, file):
+        self.file = Path(file)
+        if not self.file.exists():
+            raise ValueError(f"file path not found: {file}")
+        stat = self.file.stat()
+        self.size = stat.st_size
+        self.file_date = stat.st_ctime
+        self.spacecraft_date = None  # TODO some file name conventions?
```

Comment:
Yea so this is related to the email to Hualin about trying to standardise the filenames they are some what standard now
```
LTP03_20200101_manual_BatchRequest.PktTmRaw.SOL.0.2021.001.11.03.21.142.SRHD@2021.001.11.03.22.268.1.xml
<              name              >.< type     >.<      req date            >@<data date           >.<part>.xml
```